estmator_project/estmator_project/templates/registration/login.html

```diff
@@ -0,0 +1,44 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="container reg_form">
+    <form class="form-horizontal" method="post" action="">
+        {% csrf_token %}
+        {{ form.as_p }}
+        <input class="btn btn-primary" type="submit" value="{% trans 'Log in' %}" />
+        <input type="hidden" name="next" value="{{ next }}" />
+    </form>
+  <p>{% trans "Forgot your password?" %} <a href="{% url 'auth_password_reset' %}">{% trans "Reset it" %}</a>.</p>
+</div>
+
+{% endblock %}
+
+
+{% comment %}
```

Comment:
please please please remove comments like this from your templates.  It looks lazy when you don't.